biogtr/datasets/sleap_dataset.py

```diff
@@ -202,7 +219,7 @@ def get_instances(self, label_idx: List[int], frame_idx: List[int]) -> list[dict
                 poses.append(
                     dict(
                         zip(
-                            [n.name for n in instance.skeleton.nodes],
+                            [n.name.lower() for n in instance.skeleton.nodes],
```

Comment:
Why `.lower()` here? I think this one should probably be case sensitive TBH since it's a config-specifiable thing and we want to be precise.